pkg/virt-handler/migration-target.go

```diff
@@ -511,6 +521,11 @@ func (c *MigrationTargetController) execute(key string) error {
 		return nil
 	}
 
+	if !c.vmiExpectations.SatisfiedExpectations(key) {
```

Comment:
So with CCLM this expectation is never satisfied, and we don't get past this check.